packages/react-button/src/components/ToggleButton/useToggleButton.ts

```diff
@@ -35,20 +35,22 @@ export const useToggleButton_unstable = (
     // Slots definition
     root: {
       ...buttonState.root,
-      [isCheckboxTypeRole ? 'aria-checked' : 'aria-pressed']: !disabled && !disabledFocusable && checkedValue,
+      [isCheckboxTypeRole ? 'aria-checked' : 'aria-pressed']: checkedValue,
       onClick: React.useCallback(
         ev => {
-          if (onClick) {
-            onClick(ev);
+          if (!disabled && !disabledFocusable) {
+            if (onClick) {
+              onClick(ev);
 
-            if (ev.defaultPrevented) {
-              return;
+              if (ev.defaultPrevented) {
```

Comment:
How would `ev.preventDefault()` have been called? In the `onClick()` function?

Reply:
Yes, if it was passed down as part of the `onClick` function.